scout/server/utils.py

```diff
@@ -133,6 +133,35 @@ def institute_and_case(store, institute_id, case_name=None):
     return institute_obj
 
 
+def user_cases(store, login_user):
+    """Returns all institutes with case count for a logged user
+
+    Args:
+        store(scout.adapter.mongo.base.MongoAdapter)
+        login_user(werkzeug.local.LocalProxy)
+
+    Returns:
+        pymongo.Cursor
+    """
+
+    match_query = {
+        "$match": {
+            "collaborators": {"$in": [login_user.institutes]},
```

Comment:
Looks good, but needs an admin exception, like in https://github.com/Clinical-Genomics/scout/blob/4f210fbd15dcc8ad53d9fbd1bb29b59347b96b5f/scout/server/utils.py#L136

Reply:
Does it? As an admin using this code you should have access to all cases of all institutes anyway. And you'd get a feeling of how much times it takes for a non-admin user with many cases in many institutes

Reply:
Ah no, you are right, if you are not a collaborator then you don't get that institute. Will fix!